### ISSUE TITLE
[Website]: Screen reader is not reading the name when focus is on “Bot Framework, Microsoft teams” link present under “AdaptiveCards” main section.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6423]
### Target Platforms

NodeJS

### SDK Version

https://adaptivecards.io/  9/8/2021

### Application Name

Website

### Problem Description

[ADO 35894181](https://microsoft.visualstudio.com/OS/_workitems/edit/35894181)

**Repro Steps:**
1. Open above URL with valid credential.
2. Turn on the Narrator using ctrl+win+enter key.
3. Navigate to 'Bot Framework, Microsoft Teams and other links' and observe the issue.

**Actual Result:**
Screen Reader does not read the names of the links i.e. Bot Framework, Microsoft links, Outlook actionable messages and Window Timeline.
It is announcing as 'Link product logo'. 

**Expected Result:**
Screen Reader should convey the name for the links and it should announce as 'Bot Framework link'.

### Card JSON

```JSON
No card
```

**Comment by paulcam206** [comment]
This has been fixed.
